.github/workflows/src/set-status.js

```diff
@@ -58,33 +71,34 @@ export async function setStatusImpl({
   target_url,
   github,
   core,
+  monitoredWorkflowName,
+  requiredStatusName,
+  overridingLabel,
 }) {
-  const statusName = "[TEST IGNORE] Swagger Avocado";
-
   // TODO: Try to extract labels from context (when available) to avoid unnecessary API call
```

Comment:
Label extraction from context is possible, but the context for the workflow isn't guaranteed to be the latest. It's worth the API hit.